third_party/move/tools/move-linter/src/model_ast_lints.rs

```diff
@@ -8,6 +8,7 @@ mod needless_bool;
 mod needless_deref_ref;
 mod needless_ref_deref;
 mod needless_ref_in_field_access;
+mod nonminimal_bool;
```

Comment:
```suggestion
mod non_minimal_bool;
```

Reply:
(apply the change correspondingly on filenames etc)

Reply:
Note that current name (``nonminimal_bool``) was picked because it matches name in Clippy lint (https://rust-lang.github.io/rust-clippy/master/index.html#nonminimal_bool). I don't have a strong preference for name, but I thought it might be good to match Clippy.

Reply:
Ah, I see. I made this comment because the corresponding checker was named `NonMinimalBool`, so was suggesting the snake version (we don't want to go back after we pick the name, due to backward compat).

Picking the Rust-naming style sounds good.